plugin/column-filter/column.filter.view.js

```diff
@@ -40,6 +40,12 @@ export class ColumnFilterView extends PluginView {
 		return !this.stateAll() && (this.items.some(this.state.bind(this)) || this.byBlanks);
 	}
 
+	isEmpty() {
+		const by = Array.from(this.by);
```

Comment:
Do by has length property itself? If it is Set?

Reply:
no, Set.length always returns 0

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set/size
use size prop